src/common/boilerplates.ts

```diff
@@ -0,0 +1,6 @@
+export const boilerplates = {
+  juggernaut: {
```

Comment:
Are you guys using juggernaut? [This](https://github.com/maccman/juggernaut) Juggernaut?

Reply:
Never mind... Just realised you guys have named your boilerplate after a fucking old tool.

Reply:
Super relieved... I can't even express it in words.

Reply:
Pablooo!!! Come back to us.